app/renderer/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import "./Slider.scss";
+
+type SliderProps = {
+  max: string;
+  value: string;
+  children?: React.ReactNode;
```

Comment:
do we need this one now?

Reply:
seven's line or this every initialization with types?